webclient/src/containers/Layout/Layout.tsx

```diff
@@ -0,0 +1,20 @@
+
+import { Header } from 'components';
+
+function Layout(props:LayoutProps) {
+    const { children, className, showNav = true } = props;
+    return (
+        <div className={className}>
+            <Header showNav={showNav} />
```

Comment:
@seavor this line raises the TS the error below.  Any insight?

```
Type '{ showNav: boolean; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<Pick<RouteComponentProps<any, StaticContext, unknown>, never>, any, any>> & Readonly<...> & Readonly<...>'.
  Property 'showNav' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<Pick<RouteComponentProps<any, StaticContext, unknown>, never>, any, any>> & Readonly<...> & Readonly<...>'.ts(2322)
```

Reply:
Hmmm, not without playing with it. I'm not sure ive ever tried assigning default values within a deconstructor  though.

Reply:
Oh, probably because you're trying to assign a value to a readonly prop property.

Reply:
I'll move this to a `defaultProp` and see how it responds.  Thanks

Reply:
Nope, that didn't do it.  I was able to figure it out, though.  It's the use of `withRouter`; it doesn't play nice with passing down props.  I'll take a swing at converting this component to a functional component so we can take advantage of `useRouter`.